tests/attention/test_vsa_block_sparse_sm120.py

```
@@ -0,0 +1,459 @@
+    return mask
+
+
+def _pytorch_ref(
+    q: torch.Tensor,   # [M, H, D]
+    k: torch.Tensor,   # [N, H, D]
+    v: torch.Tensor,   # [N, H, D]
+    indptr: torch.Tensor,
+    indices: torch.Tensor,
+    R: int,
+    C: int,
+    sm_scale: float = None,
```

Ruff (0.16.0)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)